package ru.edu.hse.data.gpt.exceptions

import ru.edu.hse.common.UserFriendlyException

class ChatGPTException: UserFriendlyException("Ошибка подключения к ChatGPT.")